### PR TITLE
 Git,Linux and Scrum Basics file 1

### PR DESCRIPTION
**Pull request by pradeepm2406**
These files are the first installment of git ,linux and scrum basics

### CODE REVIEW

**Review by kulkarni90 — COMMENTED** [review]
So far so good,
1.please add  more git commands as much as we learnt till date.
2.same with linux commands too.
3.scrum intro has much scope to add
ex: participants in scrum,scrum event,Artifacts.

please clarify if you want  to  give only this inputs.

**Comment by pradeepm2406** [comment]
1. More git commands will be added in the next branch pradeep2
2. more linux commands coming up in next branch pradeep2
3. additional scope will be added in the next branch pradeep2

File: git.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- It is designed for cordinating work among progammers,but it can be used to track changes in any sets of files.
+- Git maintains a history of all the changes performed on software code by various developers.
+- Git provides features like branches and merges for developers to work on various features independently and later merge with the master.
+- git --version = returns the current version of git installed on system
+- git init  = adds a local Git repository to the project
+- git add <file> = stages the specified file
+- git add file1 file2 file3 = adds multiple files
+- git add . = adds all the files and folder to the staging area
+
+
```
Comment (david1412) [inline git.txt:15]: I don't see 10 commands here. Please write the commands

File: linux.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ shift +i = insert before cursor
+ esc, :wq or :q = save file and close
+ esc :1,$d = deletes all lines
+ dd or :d = Delete current line
+ vi <file-name> = creates and open a file
+ pwd = present working directory
+ cd <file name> = create directory
+ cd .. = go up one folder level
+ esc = terminate insert command
+
```
Comment (david1412) [inline linux.txt]: Remove the spaces below from line 12. More explanation needed here.

File: scrum.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- It is a framework rather than a methodology for developing complex products.
+- The actvities are planned and executed in time-boxed intervals known as sprints.
+- The framework works on the basis of iterative feedback loop and incremental development in time-boxed sprints for managing software/prodcut development.
+- the framework defines the roles,events,artifacts and all the underlying rules that connect them together
+- communication is given more important in scrum as information has to be passed effectively and with transparently
+- collaboration is key for the success of scrum as employees from varied background work together to build a efficient product.
+- functioning software is one key aspect which means that at any block of time, the software is in functioning state which is not in the case of waterfall methodology.
+- teams are self organized  and cross-functional and hence there is no project manager to lead and direct the teams
+- adapatation to new business developments is easier as the framework is independent of the software/product.
+
```
Comment (david1412) [inline scrum.txt:12]: Artefacts are missing ( minimum of 3-5 needed) for example, what is sprint planning, sprint backlog, daily scrum. etc

**Review by david1412 — COMMENTED** [review]
Please see the comments. The content is missing.

File: scrum.txt (hunk 3 above)
Comment (pradeepm2406) [reply scrum.txt:12]: 1. I started with the introduction of git . The commands will follow in the file2 in pradeep2 branch.
2. I have done the requested modification and added more explination
3. info on scrum artefacts will follow in the coming files in pradeep3 and pradeep4 branches

File: git.txt (hunk 1 above)
Comment (prudhvidhargudimalla) [reply git.txt:15]: add few more commads

Comment (pradeepm2406) [reply git.txt:15]: I have added the  additional git commands in the file2 in pradeep2 branch.

**Review by prudhvidhargudimalla — COMMENTED** [review]
good but why don't you start explaining commands from basic

**Comment by pradeepm2406** [comment]
Can you be more specific about which file you want to comment.

File: git.txt (hunk 1 above)
Comment (pradeepm2406) [reply git.txt:15]: please approve in case you do not have any more comments

**Comment by pradeepm2406** [comment]
If there are no comments, please approve the pull request

**Review by pradeepm2406 — COMMENTED** [review]
Have added more explanation and removed the empty lines.

**Review by pradeepm2406 — COMMENTED** [review]
Done the requested changes.

**Review by pradeepm2406 — COMMENTED** [review]
I have started wit the basics on all the topics. Any missing point will be added in the upcoming files

File: git.txt (hunk 1 above)
Comment (nithindeva) [reply git.txt:15]: What do we call the stored code in git please explaina and update.

File: scrum.txt (hunk 3 above)
Comment (nithindeva) [reply scrum.txt:12]: please follow semantic versioning.

**Comment by pradeepm2406** [comment]
Done the modifications as per Nithin's comments
1. Explained about the source code in git.txt file.
2. Modified  the scrum.txt with semantic versioning

File: git.txt (hunk 1 above)
Comment (pradeepm2406) [reply git.txt:15]: Done the modifications, accordingly. Please review and confirm

File: scrum.txt [hunk 4]
```diff
@@ -3,11 +3,11 @@ SCRUM INTRODUCTION:
...
-- communication is given more important in scrum as information has to be passed effectively and with transparently
-- collaboration is key for the success of scrum as employees from varied background work together to build a efficient product.
-- functioning software is one key aspect which means that at any block of time, the software is in functioning state which is not in the case of waterfall methodology.
-- teams are self organized  and cross-functional and hence there is no project manager to lead and direct the teams
-- adapatation to new business developments is easier as the framework is independent of the software/product.
+- The framework defines the roles,events,artifacts and all the underlying rules that connect them together
+- Communication is given more important in scrum as information has to be passed effectively and with transparently
+- Collaboration is key for the success of scrum as employees from varied background work together to build a efficient product.
+- Functioning software is one key aspect which means that at any block of time, the software is in functioning state which is not in the case of waterfall methodology.
```
Comment (david1412) [inline scrum.txt:9]: Indent missing

**Review by david1412 — COMMENTED** [review]
Left comments

File: scrum.txt (hunk 4 above)
Comment (david1412) [reply scrum.txt:9]: Good.

**Review by david1412 — COMMENTED** [review]
One change to commit, Then I'll approve PR

**Review by david1412 — APPROVED** [review]
Looks good

**Review by kulkarni90 — APPROVED** [review]
nice work brother.But do add more about scrum in next task.

**Review by nithindeva — APPROVED** [review]

**Pull request merged**